sixtyfps_runtime/corelib/model.rs

```diff
@@ -263,7 +263,9 @@ impl<T> Model for ModelHandle<T> {
     }
 
     fn attach_peer(&self, peer: ModelPeer) {
-        self.0.as_ref().map(|model| model.attach_peer(peer));
+        if let Some(model) = self.0.as_ref() {
```

Comment:
This comes from clippy, right?

Reply:
Yes: https://rust-lang.github.io/rust-clippy/master/index.html#option_map_unit_fn

I had similar changes in other places before.

Reply:
Thanks. Yeah I agree with the change, I also do find it slightly more readable:)